netmanager/src/views/pages/DeployDevice/index.js

```
@@ -0,0 +1,552 @@
+    powerType && mountType && height && height > 0 && deviceName && !deviceNameError;
+  const isStep1Valid =
+    latitude && longitude && siteName && !latitudeError && !longitudeError && !siteNameError;
+
+  const validateCoordinate = (value, setter) => {
+    if (value === '') {
+      setter('');
+      return;
+    }
+
+    const numValue = parseFloat(value);
+    if (isNaN(numValue)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)